src/cli/mod.rs

```diff
@@ -374,35 +295,20 @@ impl ReplCli {
                 repl.start()
             }};
         }
-        let config = get_config(&self.config)?;
-        tracing::info!("Configured variables: {:?}", config);
         set_lurk_dirs(
-            &config,
             &self.public_params_dir,
             &self.proofs_dir,
             &self.commits_dir,
             &self.circom_dir,
         );
-        let rc = get_parsed_usize("rc", &self.rc, &config, DEFAULT_RC)?;
-        let limit = get_parsed_usize("limit", &self.limit, &config, DEFAULT_LIMIT)?;
-        let backend = get_parsed(
-            "backend",
-            &self.backend,
-            &config,
-            parse_backend,
-            DEFAULT_BACKEND,
-        )?;
-        let field = get_parsed(
-            "field",
-            &self.field,
-            &config,
-            parse_field,
-            backend.default_field(),
-        )?;
+        let rc = LURK_CONFIG.reduction_count;
+        let limit = LURK_CONFIG.limit;
+        let backend = &CLI_CONFIG.backend;
+        let field = &CLI_CONFIG.field;
         validate_non_zero("rc", rc)?;
-        backend.validate_field(&field)?;
-        match field {
-            LanguageField::Pallas => repl!(rc, limit, pallas::Scalar, backend),
+        backend.validate_field(&field.0)?;
+        match field.0 {
+            LanguageField::Pallas => repl!(rc, limit, pallas::Scalar, backend.clone()),
```

Comment:
It should be possible to downstream just `backend` and make consumers not require ownership

Reply:
The `Repl::new()` function requires an owned `Backend`, we could instead borrow it with an explicit lifetime but for such a simple enum I don't thik the clone matters.